exercises/practice/grade-school/grade_school.py

```diff
@@ -10,3 +10,6 @@ def roster(self):
 
     def grade(self, grade_number):
         pass
+
+    def added(self):
+        pass
```

Comment:
```suggestion
        pass
        
        
    def add_student(self, name, grade):
        pass
```

Reply:
Thanks for responding @BethanyG ! I'm not sure I understand your suggestions. The existing stub file,  before my changes, is below, and it starts with `class School` and includes `def add_student(self, name, grade)`. So unless I'm missing something, there's no need for the changes you propose. Am I missing something?

```
class School:
    def __init__(self):
        pass

    def add_student(self, name, grade):
        pass

    def roster(self):
        pass

    def grade(self, grade_number):
        pass
```